.github/workflows/dapr-ambient.yaml

```diff
@@ -78,6 +78,55 @@ jobs:
           name: ambient-init
           path: ${{ env.ARCHIVE_OUTDIR }}
 
+  test:
+    name: "Test artifacts on Kind cluster"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v3
+
+      - name: Install kubectl
+        run: |
+          curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+          sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+          kubectl version --client --output=yaml
+
+      - name: Create k8s Kind Cluster
+        uses: helm/kind-action@v1.5.0
+      
+      - name: Setup helm
+        run: |
+          curl https://baltocdn.com/helm/signing.asc | gpg --dearmor | sudo tee /usr/share/keyrings/helm.gpg > /dev/null
+          sudo apt-get install apt-transport-https --yes
+          echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/helm.gpg] https://baltocdn.com/helm/stable/debian/ all main" | sudo tee /etc/apt/sources.list.d/helm-stable-debian.list
+          sudo apt-get update
+          sudo apt-get install helm
+
+      - name: Install dapr chart
+        run: |
+          kubectl config use-context kind-chart-testing
+          helm repo add dapr https://dapr.github.io/helm-charts/
+          helm repo update
+
+          helm upgrade --install dapr dapr/dapr \
+          --version=1.11 \
```

Comment:
For now we should probably remove the version and always use the latest version.